src/js/api/global-api.js

```diff
@@ -33,7 +34,9 @@ define([
             return _addPlayer(new Api(domElement, _removePlayer));
         }
         // invalid query
-        return null;
+        return {
+            registerPlugin: plugins.registerPlugin
```

Comment:
Based off of the usage, it appears to me that registerPlugin should be moved from the jwplayer instance API to the global API. 

ie, from ```jwplayer().registerPlugin``` to ```jwplayer.registerPlugin```

What do you think?